webhook/go.mod

```diff
@@ -0,0 +1,46 @@
+module webhook-go
```

Comment:
You should move this file to the top level folder. It'll make things easier later. I added one on the main branch already, just overwrite it with yours.

Reply:
Does that include `main.go`? Just moving `go/mod` seems to break everything.

```
$ go build
no Go files in /.../github_actions_on_gcp
$ go build webhook/.
package webhook is not in std (/usr/lib/google-golang/src/webhook)
```

Reply:
I figured it out.